app.js

```diff
@@ -18,22 +18,36 @@ const express = require('express');
 const statusCodes = require('http-status-codes');
 
 const errorTracker = require('./routes/error-tracker');
-const querystring = require('./utils/requests/query-string');
 const parseErrorHandling = require('./utils/requests/parse-error-handling');
 
 const app = express();
-const jsonParser = express.json({
-  limit: 10 * 1024 /* 10kb */,
-  type: () => true, // Attempt to allow any content-type.
-});
+const BODY_LIMIT = 10 * 1024 /* 10kb */;
+function rawJsonBodyParserMiddleware(req, res, next) {
+  if (!req.rawBody) {
+    // Defer to bodyParser when running as a server.
```

Comment:
Do we even need this anymore?

Reply:
Probably not tbh. We could probably get rid of the server module entirely, if we have no plans to redeploy to app engine

Reply:
Oh now I remember; the e2e tests are written using the express app as a standard Express server, and I haven't yet spent the time to rewrite the tests to use whatever test harness the Cloud Functions library (hopefully) provides

Reply:
I have no plans to app engine. Given that it's failed miserably during deploy, I'd consider it dead.

Reply:
I think we had a race condition on my last comment. I'd like to punt on this for now and rewrite the e2e tests in a follow-up

Reply:
That's fine. I don't imagine this will be difficult, we just need to install an extra `app.use()` helper in the test setup.